frontend/pages/index.vue

```diff
@@ -1,3 +1,132 @@
 <template>
-  <h1>Home Page</h1>
+  <v-container fluid>
+    <v-row>
+      <v-col cols="12">
+        <h1 class="text-h4 ml-4"> Dashboard </h1>
```

Comment:
Instead of this in the page itself, what about moving this top bar to the layout and just having the application name (e.g. "OEE SMIP App")?